app/assets/stylesheets/components/header.scss

```diff
@@ -93,6 +94,7 @@
   height: 100%;
   display: flex;
   align-items: center;
+}
 
   @media (min-width: $breakpoint-s) {
     perspective: 1000px;
```

Comment:
TIL about the [perspective](https://developer.mozilla.org/en-US/docs/Web/CSS/perspective) CSS property.